MANIFEST.in

```diff
@@ -0,0 +1,10 @@
+recursive-include dask_sphinx_theme *.py
+recursive-include dask_sphinx_theme *.html
+recursive-include dask_sphinx_theme *.css
+recursive-include dask_sphinx_theme *.svg
+recursive-include dask_sphinx_theme *.conf
+
+include setup.py
+include README.rst
+include LICENSE.txt
+include MANIFEST.in
```

Comment:
This gets added automatically. So doesn't need to be explicitly listed. Though it shouldn't hurt to have it.